Safety/Safety-Systems/GIS.rst

```diff
@@ -1,6 +1,342 @@
+
+      GIS Detections and actions
+
+The status of the system is displayed, the symbols and their representation are: 
+
+* **BYP** within an orange square: Indicates the action is bypassed. It logs alarms but doesn't trigger any action.
+
+* Green square: Indicates an interlock not triggered.
+
+* Red square: Indicates a triggered interlock.
+ 
+    
+
+.. admonition:: M2 changes in 2024
+
+    * **Addition**: Signal **D-20** "M2 mirror in *closed-loop*" as a cause for the action **A-5** "Block TMA motion".
+
+    * **Removal**: Signal **D-2** “Earthquake Alarm” from **A-13** “M2 STO” action signal.
+
+
+.. _Safety-Systems-GIS-Main-Controller-Interface:
+
+Bypass and reset actions
+-------------------------
+
+* Any user can :guilabel:`RESET` a subsystem.
+
+* **Wired subsystems** are reset in GIS Main Panel. 
+    E.g M2 interlock can be reset in GIS.
+
+* **Network based systems** reset on their own in GIS main panel once the reset in done in the specific subsystem.
+     E.g. A M1M3 interlock should be reset in M1M3 IS and then it will be clear in GIS main panel.
+
+* Only admin user can bypass a subsystem.
+
+* Detections can be bypassed but actions cannot.
+
+* The D2 earthquake alarm may be bypassed at times, as it can cause the TMA to lose power if there are power or capacitor bank issues. (The acceleration threshold is approximately 0.3g, equivalent to a magnitude 5-6 earthquake).
+
+.. note::
+    
+    D4 GIS E-stop: Safety gates E-stops are hit very often, so they are currently physically bypassed. This will be undone when operations start.
+
+.. _Safety-Systems-GIS-Main-Controller-Interface:
+
+Troubleshooting
+================
+
+In order to troubleshoot you need to understand the structure and check the alarm logs, at the bottom of the Main panel interface.
+
+.. admonition:: Important
+
+    Beware that sometimes the detection disappears, this means the interlocks are no longer triggered (because someone may have reset it) but the action is still in place.
+
+
+.. _Safety-Systems-GIS-Troubleshooting-Dealing-with-interlocks-in-GIS-Main-Control-Panel:
+
+Dealing with interlocks in GIS Main Control Panel
+--------------------------------------------------
+
+Remember, depending on the :ref:`connection type <Safety-Systems-GIS-connection-type>`
+
+* **Wired subsystems** are :guilabel:`RESET` at the Main Control Panel.
+
+* **Network-based systems** require resetting within their corresponding subsystem to clear the interlock in the Main Control panel.
+
+
+.. _Safety-Systems-GIS-Troubleshooting-Dealing-with-interlocks-in-GIS-Main-Control-Panel-Top-end-systems:
+
+Top End systems
+^^^^^^^^^^^^^^^^^
+
+Top End systems** (M2CAM), are a special case. The status listed does NOT indicate the status of each individual element, but rather the **status of the E-Stop circuit** (including reset action) for each element.
+
+After an **STO or power cycle** of the Top End systems, you must :guilabel:`RESET` the GIS E-stop status buttons 
```

Comment:
I think I am confused with the E-stop status notation as I don´t see it in the screen. Do you mean the M2 ACTUATOR, M2 HEXAPOD... RESET buttons?